src/nfv_test_api/v2/services/gnodeb.py

```diff
@@ -223,25 +223,32 @@ def node_status(self, identifier: str) -> Dict[str, Any]:
         # make sure the config exists
         gnb = self.get_one(identifier)
 
-        if identifier not in self.process_handler.processes:
-            raise NotFound(f"No gNodeB process found for nci {identifier}")
```

Comment:
I think we should still raise this, as it means we are trying to get the status of a service that we either never started or we already stopped